lib/helper/request-store.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+// Use got directly because the screwriver-request does not return proper format of error message when the responseType is 'buffer'.
```

Comment:
is it possible to enhance screwdriver-request to support buffer ?

Reply:
OK, I make an issue about this, and will fix it in that issue.
Let me just add the comment for now.

Reply:
@jithine 
I made an issue.
https://github.com/screwdriver-cd/screwdriver/issues/2623